services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/directory_watcher.py

```diff
@@ -104,10 +100,18 @@ def __init__(self, loop: AbstractEventLoop, directory_path: Path):
 
         self.loop: AbstractEventLoop = loop
         self.directory_path: Path = directory_path
+        self._is_enabled: bool = True
+
+    def set_enabled(self, is_enabled: bool) -> None:
+        self._is_enabled = is_enabled
+
+    def _call_trigger_async_invoke_push_mapped_data(self) -> None:
+        trigger_async_invoke_push_mapped_data(self.loop, self.directory_path)
```

Comment:
;) too many verbs here don't you think? I am not quite sure I understand the need of a function that directly calls another?

Reply:
Hmm I still don't find it self explanatory.. maybe we can discuss this?

Reply:
trigger async invoke push mapped data: 3 actions

Reply:
I made it with less now 😄, also function names are now private